### DISCUSSION TITLE
How do I manually start the payout process?

### DISCUSSION THREAD

**Discussion by kr1z1s** [coinfoundry/miningcore#1211] [Q&A]
Hello!

Due to the fact that the uncle block was registered in the database as orphan, payments did not start. I changed the block in the database to confirmed but payments did not start.

How do I manually start the payout process?

ETH pool

**Comment by oliverw** [comment]
"The" uncle block. Are we supposed to know "the" block??

**Reply by kr1z1s** [reply]
> "The" uncle block. Are we supposed to know "the" block??
I have same error https://github.com/oliverw/miningcore/issues/633

I mined uncle https://etherscan.io/address/0xbc5d8ccedc79812ec93ba1cf63000662ca3e09c8#uncle
https://etherscan.io/uncle/0x66bae1a16b850ace90ab4f6773c17ebd24e8b4581d9e41eebce83951e5c33b41

On my pool this block marked as orphan.

I'm from inside DB changed fields 'status', 'reward','type' and 'confirmation'

But payouts process did not start

**Reply by oliverw** [reply]
If the pool misclassified the uncle as `orphaned`, when it in fact was not you need to update its status to `pending` to get it processed.

**Reply by kr1z1s** [reply]
It worked. Thank you so much.

**Reply by oliverw** [reply]
You're welcome.

**Reply by kr1z1s** [reply]
One more question. How long before end of current epoch, generation of a new DAG begins?
And is it possible to start generation manually? For example through DB.

**Reply by oliverw** [reply]
We try to always have the DAG for the following epoch ready. So generation of next epoch should start as soon as the pool loads the current one and the next one has not been generated yet.

**Reply by kr1z1s** [reply]
> We try to always have the DAG for the following epoch ready. So generation of next epoch should start as soon as the pool loads the current one and the next one has not been generated yet.

I have eth pool. 40 gh. Several times when switching the epoch, I saw that incorrect DAG being loaded. After pool stopping, deleting all dag and starting, first DAG created correctly (30-40 min). Then a second DAG is generated. But after 5-7 min generation stoped... DAG - incorrect.

I tried deleting second DAG and restarting the pool. DAG generation started and ended after 5 minutes. I'm sure it's incorrect. It couldn't have been generated so quickly.

Tomorrow is a new epoch.  I am sure that mining will stop and I will have to delete files and restart the pool.

I think the problem is that mining is going on in parallel with generation.

:(

**Comment by jasonfifa** [comment]
I have similar issue. How can I manually start the payout process? The payment is not processed even changing the orphans block to "pending" status into the database.
